app/src/main/java/com/github/multimatum_team/multimatum/service/ProcrastinationDetectorSensorListener.kt

```diff
@@ -0,0 +1,104 @@
+package com.github.multimatum_team.multimatum.service
+
+import android.content.Context
+import android.hardware.Sensor
+import android.hardware.SensorEvent
+import android.hardware.SensorEventListener
+import android.widget.Toast
+import com.github.multimatum_team.multimatum.R
+import kotlin.math.abs
+
+class ProcrastinationDetectorSensorListener(private val applicationContext: Context) :
+    SensorEventListener {
+
+    // the service will wait for this number of detections before starting to display toasts
+    private var nonReportedDetectionsCnt = NON_REPORTED_DETECTIONS_CNT_INIT
+
+    // data relative to the last time a movement was detected
+    private var lastDetectionTimestampNanos: Long = LAST_DETECTION_TIMESTAMP_NONINIT_CODE
+    private var lastPosition: Array<Float>? = null  // null only at initialization
+
+    override fun onAccuracyChanged(sensor: Sensor?, accuracy: Int) {
+        /* Nothing to do */
+    }
```

Comment:
Could it be possible to make a test for this function ? (Just to call it to increase coverage based on codeclimate)

Reply:
Does that even count in coverage? There is no code in it!

Reply:
Infortunatly, based on codeClimate, yes